src/test/java/delta/util/TaskListTest.java

```diff
@@ -0,0 +1,224 @@
+package delta.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+import delta.task.Task;
+import delta.task.Todo;
+
+public class TaskListTest {
+    /**
+     * Returns list to be used for testing.
+     */
+    private ArrayList<Task> createTestList() {
+        ArrayList<Task> tasks = new ArrayList<>();
+        tasks.add(new Todo("test1"));
+        tasks.add(new Todo("test2"));
+        tasks.add(new Todo("test3"));
+        return tasks;
+    }
+
+    @Test
+    public void testEquivalentConstructors() {
+        assertEquals((new TaskList()).getTasks().toString(), (new TaskList(new ArrayList<>())).getTasks().toString());
+    }
+
+    @Test
+    public void testGetTasks() {
+        TaskList taskList = new TaskList(createTestList());
+
+        assertEquals(createTestList().toString(), taskList.getTasks().toString());
+    }
+
+    @Test
+    public void testGetTask() {
```

Comment:
Could the test function name be more descriptive to show what situations are being tested?